.storybook/main.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
is there a need to put the storybook config in the root of the repo ? will there be more storybooks in this monorepo ?

Reply:
I get your point, but I am not sure. The structure is how it was created by the nx generator. Do you think we should diverge from that?

Reply:
Not a blocker for me I guess, if this is how NX gets things done